scripts/rabbitmq-server-ha.ocf

```diff
@@ -1602,15 +1605,33 @@ get_monitor() {
         rc=$OCF_ERR_GENERIC
     fi
 
-    # If we are the master and healthy, check that we see other cluster members
-    # Order a member to restart if we don't see it
     if [ $rc -eq $OCF_RUNNING_MASTER ] ; then
+        # If we are the master and healthy, perform various
+        # connectivity checks for other nodes in the cluster.
+        # Order a member to restart if something fishy happens with it.
+        # All cross-node checks MUST happen only here.
+
+        partitions_report=$(partitions_report)
+
         for node in $(get_all_pacemaker_nodes); do
```

Comment:
Lets use get_alive_pacemaker_nodes_but function here instead of get_all_pacemaker_nodes. That way
 * if we use 'get_alive_pacemaker_nodes_but $THIS_PCMK_NODE', we don't need next check below
 * we can remove get_all_pacemaker_nodes function as it is not used anywhere

It seems OK, if we ignore nodes, which Pacemaker does not see. If these nodes are disconnected, they will not see our command to restart anyway.